lib/vizkit/tree_view.rb

```diff
@@ -12,21 +12,35 @@ def self.setup_tree_view(tree_view)
         tree_view.connect(SIGNAL('customContextMenuRequested(const QPoint&)')) do |pos|
             index = tree_view.index_at(pos)
             next unless index.isValid
-            item = index.model.itemFromIndex index
+            if index.model.is_a?(Qt::AbstractProxyModel)
```

Comment:
The more ruby-ish way to test for the proxy model is `respond_to?(:mapToSource)`

Reply:
model's `respond_to?` always return `false` (probably because they are not real method definitions?)..